src/index.ts

```diff
@@ -64,35 +67,81 @@ function serviceUnavailable(message: string): APIGatewayProxyResult {
   };
 }
 
+const putConsentToFirehose =
+    (cmpCookie: CMPCookie, callback: APIGatewayProxyCallback,
+     streamName: string) => {
+      fh.putRecord(
+          {
+            DeliveryStreamName: streamName,
+            Record: {Data: new Buffer(JSON.stringify(cmpCookie))}
+          },
+          (err, data) => {
+            if (err) {
+              console.log(err, err.stack);
```

Comment:
Should we return an error to the client in this case? It's tricky because if we return an OK then the user has a valid consent in the browser but we have no log of it.